modules/nf-core/ascat/tests/main.nf.test

```diff
@@ -0,0 +1,42 @@
+nextflow_process {
+
+    name "Test Process ASCAT"
+    script "../main.nf"
+    process "ASCAT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ascat"
+
+    test("ascat") {
+
+        config "./nextflow.config"
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id: 'test', single_end:false ],
+                    file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
+                    file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true),
+                    file(params.test_data['homo_sapiens']['illumina']['test2_paired_end_sorted_bam'], checkIfExists: true),
+                    file(params.test_data['homo_sapiens']['illumina']['test2_paired_end_sorted_bam_bai'], checkIfExists: true)
+                ]
+                input[1] = file(params.test_data['homo_sapiens']['illumina']['test2_allele_specific_recal'], checkIfExists: true)
+                input[2] = []
+                input[3] = []
+                input[4] = []
+                input[5] = []
+                input[6] = []
+                """
+            }
+        }
+
+        then {
+            assertAll (
+                { assert process.success },
+                { assert snapshot(process.out.versions).match("versions") }
+            )
+        }
+    }
```

Comment:
You will need to add the stub test here aswell. You can just copy the normal test and add stub to the name as well as a line saying ` options "-stub"`